tests/gzip/Makefile

```diff
@@ -0,0 +1,41 @@
+# ################################################################
+# Copyright (c) 2017-present, Yann Collet, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+# ################################################################
+
+PRGDIR = ../../programs
+VOID   = /dev/null
+
+
+.PHONY: all
+all: test-gzip-env test-helin-segv test-hufts test-keep test-list test-memcpy-abuse test-mixed test-null-suffix-clobber test-stdin test-trailing-nul test-unpack-invalid test-zdiff test-zgrep-context test-zgrep-f test-zgrep-signal test-znew-k test-z-suffix 
+	@echo Testing completed
+
+.PHONY: zstd
+zstd:
+	$(MAKE) -C $(PRGDIR) zstd
+	#alias gzip='$(PRGDIR)/zstd --format=gzip'
```

Comment:
1. `alias` doesn't work because `make` runs a new `sh`
2. `ln` works fine but a path depends on OS and I think that `ln` should be set manually (and one can remove it manually)